android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DividerViewHolder.java

```diff
@@ -0,0 +1,28 @@
+package org.fossasia.openevent.adapters.viewholders;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import org.fossasia.openevent.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by Shreyas on 6/29/2017.
+ */
+
+public class DividerViewHolder extends RecyclerView.ViewHolder {
```

Comment:
Same here, can be made an inner class of the adapter.
No special reason to create it separately.